mlflow/models/docker_utils.py

```diff
@@ -31,7 +31,9 @@
     https://github.com/pyenv/pyenv.git /root/.pyenv
 ENV PYENV_ROOT="/root/.pyenv"
 ENV PATH="$PYENV_ROOT/bin:$PATH"
-RUN apt install -y python3.7
+RUN apt install -y software-properties-common \
+    && add-apt-repository -y ppa:deadsnakes/ppa \
+    && apt install -y python3.7 python3.7-distutils
```

Comment:
If we're going to update MLflow to require Python 3.8, shall we update this as well? https://github.com/mlflow/mlflow/pull/7161/files#diff-848204af1ec28fbbde224caf1c529b15c8f432f2de419830ace7cf2eb33d6abd

Reply:
Got it, I'll update it.